apps/core/tests/test_event_loop_runner.py

```
@@ -0,0 +1,113 @@
+    raise RuntimeError(msg)
+
+
+def _failing_event_handler(context: _DummyContext) -> None:
+    msg = "event handler boom"
+    raise RuntimeError(msg)
+
+
+class _CaptureHandler(logging.Handler):
+    """Collect emitted log records for later inspection."""
+
+    def __init__(self):
```

Ruff (0.14.6)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)